code/modules/clothing/clothing_accessories.dm

```diff
@@ -132,13 +132,20 @@
 		return
 	var/obj/item/clothing/accessory/A
```

Comment:
Please rename this to `picked_accessory` or `accessory` or something like that like you have in human_stripping. Also consider just making most of this a dedicated proc like `pick_remove_accessory` since you have the same code in two places.